utils/src/command.rs

```diff
@@ -84,6 +88,19 @@ impl CommandRunExt for Command {
         self.status()?.check_status(stderr)
     }
 
+    #[allow(unsafe_code)]
+    fn lifecycle_bind(&mut self) -> &mut Self {
+        // SAFETY: This API is safe to call in a forked child.
+        unsafe {
+            self.pre_exec(|| {
+                rustix::process::set_parent_process_death_signal(Some(
```

Comment:
Oh, that's real nice.  If you wanna do that from Python you're gonna be ctyping your way into `prctl()` with some hardcoded constants...